src/components/chatInput.tsx

```diff
@@ -43,11 +43,11 @@ export function ChatInput({ onSubmit, currentQuestionType }: ChatInputProps) {
           handleSubmit(event)
           event.preventDefault()
         }}
-        className="flex w-full items-end gap-1.5 sm:gap-4"
+        className="flex w-full items-end gap-1.5 bg-background sm:gap-4"
       >
         <Textarea
           ref={textareaRef}
-          className="max-h-52 resize-none px-3 py-[7px] sm:ml-4 md:ml-8 lg:ml-20"
+          className="max-h-52 resize-none px-3 py-[7px] sm:ml-4 md:ml-8 lg:ml-20 "
```

Comment:
```suggestion
          className="max-h-52 resize-none px-3 py-[7px] sm:ml-4 md:ml-8 lg:ml-20"
```

It doesn't hurt anything, but I think we don't need this space here